core/meta/src/TListOfEnums.cxx

```diff
@@ -183,6 +183,22 @@ TEnum *TListOfEnums::Find(DeclId_t id) const
    return (TEnum *)fIds->GetValue((Long64_t)id);
 }
 
+////////////////////////////////////////////////////////////////////////////////
+/// Specialize FindObject to do search for the
+/// a enum just by name or create it if its not already in the list
+
+TObject *TListOfEnums::FindObject(const char *name) const
+{
+   TObject *result = THashList::FindObject(name);
+   if (!result) {
+      TInterpreter::DeclId_t decl;
+      if (GetClass()) decl = gInterpreter->GetEnum(GetClass(), name);
+      else            decl = gInterpreter->GetEnum(nullptr, name);
+      if (decl) result = const_cast<TListOfEnums *>(this)->Get(decl, name);
```

Comment:
Eew. Can we make this function non-const?

Reply:
Not without being backward incompatible (We would probably need to expose this type rather than its base).   Note: this is existing code that is just moved around.